src/Symfony/Component/Serializer/Mapping/Loader/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd

```diff
@@ -81,4 +84,17 @@
         <xsd:attribute name="ignore" type="xsd:boolean" />
     </xsd:complexType>
 
+    <xsd:complexType name="context">
+        <xsd:choice maxOccurs="unbounded">
+            <xsd:element name="group" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="value" type="context_value" />
+        </xsd:choice>
+    </xsd:complexType>
+
+    <xsd:complexType name="context_value">
+        <xsd:sequence>
+            <xsd:any processContents="lax" maxOccurs="unbounded" />
```

Comment:
Why not using a recursive type here? Like this:
```php
    <xsd:complexType name="context-attribute" mixed="true">
        <xsd:sequence minOccurs="0">
            <xsd:element name="attribute" type="context-attribute" maxOccurs="unbounded" />
        </xsd:sequence>
        <xsd:attribute type="xsd:string" name="name" />
    </xsd:complexType>
```

Reply:
Something like this? A bit more and less verbose at the same time 😄

Reply:
I think it's better than being lax!